dids/diddht/internal/pkarr/relay.go

```diff
@@ -0,0 +1,123 @@
+package pkarr
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/tbd54566975/web5-go/dids/diddht/internal/bep44"
+)
+
+type PkarrRelay struct {
+	relay  string
+	client *http.Client
+}
+
+func NewPkarrRelay(relay string, client *http.Client) *PkarrRelay {
```

Comment:
`NewClient` maybe?

Reply:
yea, I thought about that after I pushed. changing